### DISCUSSION TITLE
GNUStep

### DISCUSSION THREAD

**Discussion by andrewcrook** [ravynsoft/ravynos#194] [Ideas]
Might be worth looking at GNU's implementation of OpenNeXT it has objectC development tools and iirc a Cocoa API implementation. It does not have Swift support yet but Swift is open sourced now.

I wonder if getting openGNU ported to BSD might be a start towards cocoa api support?

http://www.gnustep.org

More info and links here including links to ported software.

**Comment by mszoek** [comment]
Thanks @andrewcrook ... We do actually have some Cocoa support today (not from GNUstep; it is from a project called Cocotron which is also the source of Darling's Cocoa). I'm pretty familiar with GNUstep and have been in touch with the lead developer there. We'll see where that goes.

**Reply by asvitkine** [reply]
Had to search the GitHub discussion to get details about your Cocoa impl. Perhaps it would be worth mentioning Cocotron in the FAQ under "Why aren't you working with project XXXX? Isn't this a duplication of effort?"?
